src/main.rs

```diff
@@ -75,8 +75,12 @@ async fn execute_subcommand(args: CommandLineArguments) -> ResultWithDefaultErro
                 ContinueCommand::execute(get_default_api_client()?, picker).await?
             }
 
-            List { number, entity } => {
-                ListCommand::execute(get_default_api_client()?, number, entity).await?
+            List {
+                number,
+                entity,
+                json,
+            } => {
+                ListCommand::execute(get_default_api_client()?, number, entity, Some(json)).await?
```

Comment:
If we already have a bool here, I think it's nicer if we keep that instead of wrapping it an `Option`
and then unwrapping inside `execute`.